__tests__/checkin.test.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import { render, fireEvent, screen, waitFor } from '@testing-library/react'
+import MockDate from 'mockdate'
+import fetchMock from 'fetch-mock' // eslint-disable-line import/default
+import { decrypt } from '../lib/crypto'
+import { withTestRouter } from '../config/with-test-router'
+import Checkin from '../pages/checkin'
+
+const privateKey = 'KVITtyAWMy0fKTRyqTAm2r05+FkyibXVSnlsI5v0XqQ='
+const publicKey = '5ki/YAX91GQ0ABSyBTsOXBO7tBl6ZJat+OzxnbZCjVM='
+const areaId = '5ac34aab-81f8-4f4d-bc24-97ba8d21eb7b'
+
+const route = {
+  pathname: '/checkin',
+  query: { a: areaId, k: publicKey },
+  replace: jest.fn(),
+}
+
+beforeEach(() => {
+  MockDate.set('2020-05-11T12:30:00.000Z')
+})
+
+it('shows onboarding the first time', async () => {
```

Comment:
should we also have a test for the old guest data missing address?

Reply:
Yes! We would also need a test that the user data gets remembered at all, if the checkbox is checked... and similar tests like that. I would suggest that we add them in a follow-up PR.

Reply:
Agreed, great that we have a test for this for a start.